src/driver/shader.rs

```diff
@@ -7,8 +7,8 @@ use {
     log::{debug, error, trace, warn},
     ordered_float::OrderedFloat,
     spirq::{
-        ty::{ScalarType, Type},
-        DescriptorType, EntryPoint, ReflectConfig, Variable,
+        prelude::*,
```

Comment:
Before we commit this I'll expand the star into all required types just because the rest of the library has been explicit about these so far.